### ISSUE TITLE
Convert M4B to AAC

### ISSUE THREAD

**Issue by wrampy** [rmcrackan/Libation#429]
I would suggest to have the option to convert the M4B to AAC. AAC has a much wider support than M4B.

As far as I know the M4B contains AAC encoded audio, so this would not need re-encoding the file (contrary to the MP3 conversion). Add the chapter details to the metadata if possible.

**Comment by rmcrackan** [comment]
Thanks for the suggestion.

tl;dr: this one is unlikely to be added

The long version:

The M4B filetype wasn't a choice per se. It is simply the decrypted file. My original intent was to simply remove encryption. Once decrypted, users have the freedom to do anything they want with their files, including using another tool to convert them. My philosophy is: I want to do what I do best (get data from audible and decrypt their specialty encryption) and let other software do what it does best. for example: conversion. For years I resisted adding an mp3 conversion feature. Since it was so heavily requested, I eventually relented and added that as a convenience. I doubt that any other conversion tools will be added.

**Comment by CharlieRussel** [comment]
For what it's worth, the best audio converter I've found to date is [dbPowerAmp](https://www.dbpoweramp.com/). No, it's not free. But it uses multiple cores to speed the process of converting multiple files and it does FROM virtually every format TO virtually every format. It's also well maintained and well supported, and works on both Windows and Mac.
